06-files-input-and-output/remove_files.py

```diff
@@ -2,33 +2,43 @@
 
 import os
 
-def find_files_in_folder(path, extention, file_max_size):
+def find_files_in_folder(path, extension, file_max_size):
+    print(
+        "Searching for files with .{} extension in \"{}\" and maximum size of {}bytes".format(
+            extension, path, file_max_size
```

Comment:
I dont see that in the styling guide, only where i need to call the function

```python
the_best_function_ever(
    t he_best_parameter_ever=True, 
    the_second_best_parameter_ever=False,
    the_third_best_parameter_ever=False,
)
```


this looks uglyier than `def find_files_in_folder(extension, file_max_size, path):`
```python
def find_files_in_folder(
  extension, 
  file_max_size, 
  path,
):
..
....

```